README.md

```diff
@@ -1,6 +1,44 @@
-# library_project_scripting
-A temporary repository that stores library data from ArchivesSpace as JSON files.
+# RAC Library Discovery Catalog System
 
-Includes Resource records, associated Subjects, Agents, and  Top Container records.
+A Jekyll web application which provides the front-end user interface for the online discovery of bibliographic materials at the Rockefeller Archive Center.
 
-The sample_data directory includes a small sample set of data that we used for testing our discovery system.
+## Dependencies
+
+* [Python 3](https://www.python.org/download/releases/3.0/)
+* [Ruby 2.5.0 or higher](https://www.ruby-lang.org/en/)
+* [Jekyll](https://jekyllrb.com/)
+* [Node.js](https://nodejs.org/en/)
+
+## Local Development
```

Comment:
This section can be shortened somewhat if you point folks to the `build` script, which wraps several of these steps in a single call.